README.md

```diff
@@ -43,6 +43,32 @@ $sendinblue = new Sendinblue();
 $sendinblue->getSmsApi()->sendSms('+33600000000', 'Your name', 'The message you want to send');
 ```
 
+Email Api usage
+-------------
+
+<?php
+
+require 'vendor/autoload.php';
+
+use Scoringline\SendinblueApi\Sendinblue;
+
+$sendinblue = new Sendinblue();
+
+$sendinblue->getEmailApi()->sendEmail(
+    ['to@example.com'=> 'to name!'],
+    ['from@yahoo.com', 'from email or name!'],
+    'Invitation',
+    'You are invited for giving test',
+    'This is the <h1>HTML</h1>'
+    ['replyto@yahoo.com', 'reply to!'],
+    ['cc@example.com' => 'cc name'],
+    ['bcc@example.com' => 'bcc name'],
+    [],
+    ['Content-Type" => "text/html; charset=iso-8859-1'],
+    ['myinlineimage1.png' => 'your_png_files_base64_encoded_chunk_data']
```

Comment:
The lib is here to help working with the api. If there is a treatment to do on files, you should do it. (here 64base encode)